atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPool.java

```diff
@@ -118,7 +118,7 @@ public void run() {
                     log.error("Failed to refresh Cassandra KVS pool. Extended periods of being unable to refresh will cause perf degradation.", t);
                 }
             }
-        }, CassandraConstants.SECONDS_BETWEEN_GETTING_HOST_LIST, CassandraConstants.SECONDS_BETWEEN_GETTING_HOST_LIST, TimeUnit.SECONDS);
+        }, config.poolRefreshIntervalSeconds(), config.poolRefreshIntervalSeconds(), TimeUnit.SECONDS);
```

Comment:
Not sure we should call this `poolRefreshIntervalSeconds()` as it seems a bit misleading -- it makes me think we're repopulating the backing connection pool every N seconds (e.g. maxLifetime In Hikari https://github.com/brettwooldridge/HikariCP#configuration-knobs-baby )